www/common/make-backup.js

```diff
@@ -276,8 +284,21 @@ define([
                         }
                     }, 50);
                 };
+                var todoLink = function () {
+                    var opts = {
+                        binary: true,
+                    };
+                    var fileName = getUnique(sanitize(rawName), '.txt', existingNames);
+                    existingNames.push(fileName.toLowerCase());
+                    var content = new Blob([fData.href], { type : "text/html;charset=utf-8" });
```

Comment:
1. To be consistent with the rest of the codebase, there shouldn’t be a space before colon (it is mostly use for ternaries).
2. The file is a text file without html information in it, its mime type should hence be set accordingly (`text/plain;charset=UTF-8`).
3. Finally, it would be better to add a “newline” character at the end of the URL, otherwise some command line scripting can behave in a unexpected way:
    - For 2 links A and B containing urls `a` and `b` respectively, downloading them gives us two files `A.txt` and `B.txt`. Running `parallel firefox <{A,B}.txt` will attempt to open the url `ab` without the newline, but indeed opens the two urls with a `\n`.

All in all, I suggest the following fix:
```js
var content = new Blob([fData.href, '\n'], { type: "text/plain;charset=utf-8" });
```